test/test_ima_verification.py

```diff
@@ -15,11 +15,11 @@
 
 # BEGIN TEST DATA
 
-ALLOWLIST = {
+IMA_POLICY_TEST = {
```

Comment:
RUNTIME_POLICY_TEST ?

Reply:
I had the impression we got rid of ima_policy word within Keylime or did I misunderstand?

Reply:
@stefanberger Not entirely. There are actually two separate concepts that were previously both referred to as "IMA policies":

- the kernel IMA policy present at `/etc/ima/ima-policy`
- the user-provided Keylime policy that determines file paths and hashes to check against (which this PR overhauls)

To avoid confusion, we decided earlier on in the PR's lifecycle to rename the Keylime user policy to "runtime policies". The highest priority is to change the terminology in any place where it might be user-facing; internal Keylime references aren't as important. I'm also not necessarily sure if some of the internal Keylime references (especially deeper in the code like in `tpm_main.py` and `tpm_abstract.py`) might be referring to the kernel IMA policy instead of the Keylime user policy. CC @mpeters - any insight? Does the Keylime verifier utilize the kernel IMA policy at all?

All that said, I've renamed the "ima_policy" terminology in the test files, since that's certainly an appropriate renaming. Happy to rename others, if they make sense.

Reply:
Update: per a discussion offline with @mpeters, Keylime likely doesn't interact with the kernel IMA policy directly, so it should be safe to rename these references to `runtime_policy`.

Reply:
> Does the Keylime verifier utilize the kernel IMA policy at all?

Afaik Keylime knows nothing about the IMA policy at all.